crates/bevy_pbr/src/extended_material.rs

```diff
@@ -230,8 +236,11 @@ impl<B: Material, E: MaterialExtension> Material for ExtendedMaterial<B, E> {
         }
     }
 
-    fn alpha_mode(&self) -> crate::AlphaMode {
-        B::alpha_mode(&self.base)
+    fn alpha_mode(&self) -> AlphaMode {
```

Comment:
This is technically a breaking change and should be documented as such.

Reply:
It's not a breaking change, because E::alpha_mode() did not exist until now.